src/screens/settings/SettingsScreen.tsx

```diff
@@ -25,6 +30,67 @@ const SettingsScreen = ({ navigation }: any) => {
   const [password, setPassword] = useState('');
   const [language, setLanguage] = useState(''); // string type in Firebase
   const [currentPassword, setCurrentPassword] = useState('');
+  const db = firebase.firestore();
+  const clientCollection = db.collection('clients');
+  const { userLanguageChange } = React.useContext(LanguageContext);
```

Comment:
This won't work since LanguageContext stores a dictionary type, and you are updating it to a string type in the handleRadio function below. Refer to explanation [here. ](https://github.com/calblueprint/siren/pull/79)

> Changing language must reflect in 2 places:

> language context sets the json language dictionary (ES, EN, VIET)
> setting client's language in Firebase.
> Context Providers are used for persisting user states in the app. They are also useful for reflecting changes that the user needs to see immediately. This means LanguageProvider only has to save the state of current language dictionary, since all static texts we see across the app use a local dictionary's translations.
> 
> For dynamic texts (texts that SIREN will change), we defer to Firebase storage. Intake form questions are the only texts subject to change. This means we store a language field for the client, which we use to query questions with based on current client's language. We save this as a string ES/VIET/EN. Though we could've saved this as apart of the Language Provider, there isn't a large need for this since intake form pulls from Firebase, not the local app language.